proxy/proxy.go

```diff
@@ -0,0 +1,19 @@
+package proxy
+
+import (
+	"os"
+	"os/exec"
+	"syscall"
+
+	"github.com/pkg/errors"
+)
+
+func Exec(command string, args []string) error {
+	agent, err := exec.LookPath("picard")
```

Comment:
@zzak @dnephin what binary name should we be using here - can we use `circlec-agent`?

Reply:
`picard` is for sure supported, my guess was until we transition the name this can be a working placeholder and update the path later?

Reply:
`/bin/picard` only work with the docker executor, machine executor uses only `circleci` so unfortunately there is no consistent name we can use right now. 

I opened a PR yesterday to make it use `circleci-agent` consistently. Once that merges we should use that.